backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -48,6 +91,167 @@ JNIEXPORT jint JNI_OnLoad(JavaVM *jvm, void *reserved) {
 
 namespace /* anonymous */
 {
+    static void *
+    buffer_reader(int fd, int flags)
+    {
+        void *r;
+
+        r = mmap(NULL, BUFFER_SIZE, flags, MAP_SHARED, fd, 0);
+        if (r == MAP_FAILED)
+            abort();
+
+        return r;
+    }
+
+    static void *
+    buffer_create(void)
+    {
+        void *r;
+        int fd;
+
+        fd = bun_memfd_create("_backtrace_buffer", MFD_CLOEXEC);
+        if (fd == -1)
+            abort();
+
+        if (ftruncate(fd, BUFFER_SIZE) == -1)
+            abort();
+
+        buffer_fd = fd;
+
+        r = buffer_reader(buffer_fd, PROT_READ|PROT_WRITE);
+
+        return r;
+    }
+
+    static int
+    request_handler(pid_t tid)
+    {
+        // TODO revisit this logic
+        bool bun_initialized = bun_handle_init(&handle, BUN_BACKEND_LIBUNWINDSTACK);
+        if (!bun_initialized) {
```

Comment:
we should be consistent with brackets. if statements above don't have brackets. Java layer has (and should have). Can we decide - do we want brackets or no?

Reply:
Will resolve, thank you!